webapp/components/browser-picker.js

```diff
@@ -0,0 +1,45 @@
+import { PolymerElement } from '../node_modules/@polymer/polymer/polymer-element.js';
+import { html } from '../node_modules/@polymer/polymer/lib/utils/html-tag.js';
+import '../node_modules/@polymer/paper-dropdown-menu/paper-dropdown-menu.js';
+import '../node_modules/@polymer/paper-listbox/paper-listbox.js';
+import '../node_modules/@polymer/polymer/lib/elements/dom-repeat.js';
+import '../node_modules/@polymer/paper-item/paper-icon-item.js';
+import './display-logo.js';
+import {ProductInfo, DefaultProducts, DefaultBrowserNames} from './product-info.js';
+
+class BrowserPicker extends ProductInfo(PolymerElement) {
+  static get is() {
+    return 'browser-picker';
+  }
+
+  static get template() {
+    return html`
```

Comment:
Yeah, see the html-tag import. It's the way of converting a string in HTML DOM easily, so it can be returned as the `template` property statically for the class.